web/templates/twig-source/incuna-surveys/form/base/proportion-field.html

```diff
@@ -20,11 +12,15 @@
                         >
                     {% block input %}
                         <div class="proportion-input" integer-field model="model[$index]" id="field.id" form="form"></div>
+                        <div
+                            aif-slider-input
+                            model="model[$index]"
+                            ceiling="100"
```

Comment:
Could we calculate and expose the `remaining` and use it to limit the `ceiling` and `width` of the slider? 

```
    ceiling="{{ model[$index] + remaining }}"
    width="{{ (model[$index] + remaining)/100 }}%"
````


![image](https://cloud.githubusercontent.com/assets/473823/23630297/830ae60c-02b2-11e7-97c3-7c80ca056f76.png)